binaries/download_binaries.py

```diff
@@ -0,0 +1,152 @@
+
+  assets: List[dict] = []
+  for end_point in github_api_end_points:
+    res = requests.get(end_point,
+                       # This header is recommended by github docs.
+                       headers={'Accept': 'application/vnd.github.v3+json'})
+    res.raise_for_status()
+    res_in_json = res.json()
+    # `res_in_json[0]` will grab the latest release.
+    assets.extend(res_in_json[0]['assets'])
+
+  download_threads: List[Thread] = []
+  downloaded_files: List[str] = []
+
+  # Loop over all the assets and download them.
+  for asset in assets:
+    download_url: str = asset['browser_download_url']
+    download_threads.append(Thread(target=download_binary,
+                                   args=(download_url, downloaded_files)))
+    # Start downloading the asset.
+    download_threads[-1].start()
+
+  # Wait until all the binaries are downloaded.
+  while(any(download_thread.is_alive() for
+            download_thread in download_threads)):
+    time.sleep(1)
+
+  return downloaded_files
+
+
+def select_binaries(binaries_names: List[str],
+                    name_contains: List[str]) -> List[str]:
+  """A non-reliable method to automate selecting which binaries goes
+  for which platform."""
+
+  # The subset of the binaries we will select.
+  selected_binaries: List[str] = []
+  for binary_name in binaries_names:
+    # If we have every substring the `binary_name` so we can select it safely.
+    if all(substring.lower() in binary_name.lower() for
+           substring in name_contains):
+      selected_binaries.append(binary_name)
+  return selected_binaries
+
+
+def main():
+  # https://api.github.com/repos/{owner}/{repo}/releases
+  releases_urls = ['https://api.github.com/repos/joeyparrish/static-ffmpeg-binaries/releases',
+                   'https://api.github.com/repos/google/shaka-packager/releases']
+
+  # Download file whose names starting with one of these strings.
+  starts_with = ['ffmpeg', 'ffprobe', 'packager']
+  binaries_names = download_github_release_binaries(streamer_binaries.__name__,
+                                                    releases_urls,
+                                                    starts_with)
+
+  # Sort out the binaries based on platform.
+  package_data = {
+      # 64-bit Windows
+      'win_amd64': select_binaries(binaries_names,
+                                   name_contains=['win',]),
+      # 64-bit Linux
+      'manylinux1_x86_64': select_binaries(binaries_names,
+                                           name_contains=['linux', 'x64',]),
+      # Linux on ARM
+      'manylinux2014_aarch64': select_binaries(binaries_names,
+                                               name_contains=['linux', 'arm64',]),
+      # 64-bit with 10.9 SDK
+      'macosx_10_9_x86_64': select_binaries(binaries_names,
+                                            name_contains=['osx',]),
+    }
+
+  # Dump `package_data` into a yaml file.
+  with open(YAML_OUTPUT_FILE, 'w') as output_file:
```

Comment:
Is platform_package_data.yaml still needed in the repo?  Should both yaml files be removed from the PR, and the generated one added to .gitignore?